core/src/context.rs

```diff
@@ -15,19 +20,46 @@ pub struct Context {
     pub logger: Arc<Mutex<Logger>>,
     pub persister: Arc<Mutex<Persister>>,
     state: Option<Arc<RwLock<State>>>,
+    pub action_channel: SyncSender<ActionWrapper>,
+    pub observer_channel: SyncSender<Observer>,
 }
 
 impl Context {
+    pub fn default_channel_buffer_size() -> usize {
+        100
+    }
```

Comment:
Please make this a constant

Reply:
@sphinxc0re doesn't the rust compiler optimise this kind of thing away under the hood?

Reply:
Yes, I believe so, but I agree with @sphinxc0re that a `const` would be clearer, more scan-able.
I don't see the advantage of this form. I would be happy to learn.